components/director/cmd/director/main.go

```diff
@@ -468,3 +503,18 @@ func webhookService() webhook.WebhookService {
 
 	return webhook.NewService(webhookRepo, uidSvc)
 }
+
+func buildScheduler(namespace string) (operation.Scheduler, error) {
+	cfg, err := rest.InClusterConfig()
+	if err != nil {
+		log.D().WithError(err).Warn("Could not start K8S Scheduler. Will use the default one")
+		return &operation.DefaultScheduler{}, nil
+	}
+	k8sClient, err := client.NewForConfig(cfg)
```

Comment:
We could add timeout like: https://github.com/kyma-incubator/compass/blob/master/components/connector/cmd/main.go#L141